janus_server/src/task.rs

```diff
@@ -359,6 +345,25 @@ impl Task {
             .rev()
             .any(|t| t == auth_token)
     }
+
+    /// Returns the [`VerifyKey`] currently used by this aggregator to prepare report shares with
+    /// other aggregators.
+    ///
+    /// # Errors
+    ///
+    /// If the verify key is not the correct length as required by the VDAF, an error will be
+    /// returned.
+    pub fn primary_vdaf_verify_key<const L: usize>(&self) -> Result<VerifyKey<L>, Error> {
+        // We can safely unwrap this because we maintain an invariant that this vector is
+        // non-empty.
+        let secret_bytes = self.vdaf_verify_keys.first().unwrap();
```

Comment:
This changes from `last()` to `first()` to determine the primary verification key. OTOH, I think that's fine--it's convention, and we currently don't know how to rotate these values anyway, so no one is relying on the convention not changing. :-)

Reply:
Yeah, I was a bit unnerved by that, but our table constraints currently enforce only one verification key. Good time to get our story straight.